src/assets/scss/app.scss

```diff
@@ -1,110 +1,7 @@
-// Updated basic colors
```

Comment:
I removed these styles in here and instead wrote instructions for how to customize styling since these styles were all LCO styles.